SAR_Training/English/Ecosystems/Exercise6-Change_Detection_Amplitude_Time_Series_Hyp3.ipynb

```diff
@@ -212,7 +213,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<font face=\"Calibri\" size=\"3\"><b>Change into the analysis directory:</b></font>"
+    "# <font face=\"Calibri\" size=\"3\"><b>Change into the analysis directory:</b></font>"
```

Comment:
Are we sure that we need the `#`? Was this an unintentional comment?

Reply:
Good catch! It is in the markdown (not a python comment) and wasn't impacting the output, but it doesn't belong. It is now removed.